### PR TITLE
Update template locations, auto-update templates

### LINKED ISSUE #1114: need to auto-sync ci-operator templates to cluster
/cc @gabemontero 
/assign

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @bbguimaraes @gabemontero 
/assign @smarterclayton 
fixes #1114

### CODE REVIEW

File: ci-operator/populate-templates.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+templates="$( dirname "${BASH_SOURCE[0]}" )/templates"
+
+for templates_file in $( find "${templates}/" -mindepth 1 -maxdepth 1 -type f -name "*.yaml" ); do
+	template_name="$( basename "${templates_file}" ".yaml" )"
+	oc create configmap "prow-job-${template_name}" "--from-file=${templates_file}" -o yaml --dry-run | oc apply -f -
```
Comment (kargakis) [inline ci-operator/populate-templates.sh]: use replace instead of apply

Comment (kargakis) [reply ci-operator/populate-templates.sh]: https://github.com/openshift/release/issues/1115

**Comment by bbguimaraes** [comment]
My tab key approves.

lgtm

**Pull request merged**